includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -2979,20 +2979,34 @@ private function determine_suggested_product_type( WC_Order $order ): string {
 
 			++$product_count;
 
+			// Capture first product's type (only used for single-product orders).
+			if ( null === $product_type ) {
+				$product_type = $product->get_type();
+			}
```

Comment:
(minor - no change necessary either way) Is the check on `null === $product_type` needed? If there's one product, `$product_type` will be set as the first product. And if there's multiple products, this function will return `'multiple'` later. Does that sound accurate? 🤔 

```suggestion
			$product_type = $product->get_type();
```

Reply:
You are right, the null check isn't strictly necessary. But I think I'll keep it, it avoids reassigning `$product_type `in multi-product orders. And also makes it clear that we only capture the first product's type.  Thanks for the suggestion though!